script/Ciel/Day 13/13_01_02_CIEL13_7B.txt

```diff
@@ -755,42 +806,49 @@ Even as an Executor, you're nothing but a victim.
 [sha:d9ffc7375696817d541a30bd1853d66aa1304fb4]{
 -- Page 270, Offset 36076.
 -- 私たちには救われる奇跡も、生きている理由も、
-There's no miracle that will save us, no reason for us to live―――
+There's no miracle that will save us,
+ there's no reason for us to live―――
 }
 [sha:17ed38c7b5ab4c99d87cdf34739b3352274e3005]{
 -- Page 270, Offset 36077.
 -- 愛される価値もない。そうでしょう、エレイシア？”」
-and no right for us to be loved. Isn't that right, Elesia?"
+And there's nothing about us to be loved.
+ Isn't that right, Elesia?"
 }
 [sha:a0da66b61442473e85e878f93005168d6b0661af]{
 -- Page 271, Offset 36078.
 -- ……そう、確かにわたしには愛される価値はない。
-...That's right. I don't deserve to be loved.
+That's right... I don't deserve to be loved.
 }
 [sha:dab896e49f72bb98fc533dbe159056f5a12f85cf]{
 -- Page 271, Offset 36079.
 -- でも奇跡を見た。たったいま、なにより尊いものを見た。
-But I saw a miracle. Just now, I saw something more precious than anything else.
+But I just witnessed a miracle.
+ A moment ago, I saw something more precious than anything else.
 }
 [sha:2aac37d0a09a82aaade710545df14337fbbe9321]{
 -- Page 271, Offset 36080.
 -- 彼には何もない。幼い頃から奪われるだけの人生だった。
-It was nothing to him. It was simply a life that was taken away during his childhood.
+It was nothing to him.
+ Since he was a child, he has lived a life plagued by loss.
 }
 [sha:d28071453e68c844c2249e20a4fbc9fbf7ca7d82]{
 -- Page 271, Offset 36081.
 -- 彼本人、奪われすぎて何があったのか、それがどういう事なのかすら気づく事さえ奪われている。
-He lost so much that even questions such as what had happened, or what it all meant, were taken away as well.
+Even the realization of all that he has lost has been stolen from him,
+ along with the thought to ask what it all means.
 }
 [sha:f765111b03fe008f25e28b28283435aec2195097]{
 -- Page 271, Offset 36082.
 -- だから、何よりもまず自分の人生を、自分の幸せを取り返さないといけない人だったのに。
-That's why he should've sought to take back his own life and happiness before anything else.
+As someone who has lost so much,
+ his overriding goal should have been to take back his own life, his own happiness.
```

Comment:
"his own life; his own happiness" or "his own life, and his own happiness" or "his own life——his own happiness"